src/Mutator/Removal/ArrayItemRemoval.php

```diff
@@ -119,7 +120,22 @@ public function mutate(Node $arrayNode): iterable
 
     public function canMutate(Node $node): bool
     {
-        return $node instanceof Node\Expr\Array_ && count($node->items);
+        if (!$node instanceof Node\Expr\Array_) {
+            return false;
+        }
+
+        if (!count($node->items)) {
```

Comment:
what do you think about `count() === 0` or `$node->items === []` as we have in other places?

Reply:
Good point.